applications/GeoMechanicsApplication/python_scripts/geomechanics_Pw_solver.py

```diff
@@ -75,15 +75,14 @@ def GetDefaultParameters(cls):
             "block_builder"              : true,
             "search_neighbours_step"     : false,
             "linear_solver_settings":{
-                "solver_type": "AMGCL",
-                "tolerance": 1.0e-6,
-                "max_iteration": 100,
-                "scaling": false,
-                "verbosity": 0,
-                "preconditioner_type": "ILU0Preconditioner",
-                "smoother_type": "ilu0",
-                "krylov_type": "gmres",
-                "coarsening_type": "aggregation"
+                "solver_type":     "amgcl",
+                "smoother_type":   "ilu0",
+                "krylov_type":     "gmres",
+                "coarsening_type": "aggregation",
```

Comment:
The precconditioner_type is missing here. Maybe it is wise to do the lower case of AMGCL as only change.

Reply:
Agreed, I reverted everything that wasn't related, just did the lower case amgcl, however then we get the following problem:
```
RuntimeError: Error: AMGCL Linear Solver : preconditioner_type is invalid!
Currently prescribed preconditioner_type : ILU0Preconditioner
Admissible values are :
    amg
    dummy
    relaxation
```

So I changed that to amg